definitions/infra-container/definition.yml

```diff
@@ -3,25 +3,38 @@ type: CONTAINER
 
 synthesis:
   rules:
+  # Dockerstats receiver from otel collector.
   - identifier: container.id
     name: container.name
     encodeIdentifierInGUID: true
-    # This is a temporary condition to restrict synthesis for a particular set of telemetry data.
     conditions:
-      - attribute: nr.entity_type
-        value: container
+      - attribute: eventType
+        value: MetricRaw
+      # All metrics from the receiver starts with the 'container.' prefix.
+      - attribute: metricName
+        prefix: container.
+      - attribute: instrumentation.provider
+        value: opentelemetry
+      # This filters only metrics comming from docker receiver, given that metrics
+      # could deffer between different runtime receivers. 
+      - attribute: otel.library.name
+        value: otelcol/dockerstatsreceiver
     tags:
+      # Default resource attributes
       container.runtime:
-      container.image.name:
-      container.image.tag:
+      container.image.name:      
       container.hostname:
+      # Optional resource attribute
       container.image.id:
-      # This tag is not currently used by OTel docker stats receiver.
-      container.state:
```

Comment:
In case we finally remove it, should we remove it as well from goldenTags?

```yaml
goldenTags:
- environment
- container.state

Reply:
I think that is used by the other sources of this entity (k8s, nri-docker)